packages/components/stories/icon/icons.js

```diff
@@ -180,6 +180,7 @@ export const icons =
         'IcCashierNoAds',
         'IcCashierNoBalance',
         'IcCashierNoOrders',
+        'IcCashierNormalAd',
         'IcCashierOffRamp',
```

Comment:
why are we not importing the icon block trade ad through here?

Reply:
it seems icon with a gradient background wont work like normal icons. so those icons are to be imported separately. and that is the issue the block trade icon was not woking properly